apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -147,11 +154,42 @@ static void ble_operation_notification_cb(ble_client_ctx *ctx, ble_attr_handle a
 	return;
 }
 
+void restart_server(void) {
```

Comment:
line 163, 172, if ret is negative value, it doesn't have to return immediately?

Reply:
It should made the changes